### PR TITLE
fix: Added onKeyDown function for input field 

### LINKED ISSUE #6653: On pressing enter should trigger `create workspace` action
### What is the issue?
When a user creates a new workspace, a modal pops up where the user enters the workspace name and clicks on the `create workspace` button to create a new workspace.
Ideally, The user should also be able to create the new workspace by pressing the enter key after entering the new workspace name.

### How to reproduce the issue?

1. Go to ToolJet dashboard
2. From workspaces dropdown in the bottom, select create new workspace  
3. On the modal, enter the workspace name and press enter

### Screenshots or Screencast
image (11)

### PR DESCRIPTION
**Pull request by coderSuren**
Fixes #6653 

Added onkeyDown functionality for input field, which triggers create workspace button on pressing Enter key

### CODE REVIEW

File: frontend/src/_components/OrganizationManager/CreateOrganization.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React, { useState, useRef } from 'react';
```
Comment (kavinvenkatachalam) [inline frontend/src/_components/OrganizationManager/CreateOrganization.jsx:1]:
```suggestion
import React, { useState } from 'react';
```

File: frontend/src/_components/OrganizationManager/CreateOrganization.jsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ export const CreateOrganization = ({ showCreateOrg, setShowCreateOrg }) => {
   const [isCreating, setIsCreating] = useState(false);
   const [newOrgName, setNewOrgName] = useState('');
   const [errorText, setErrorText] = useState('');
+  const createButtonRef = useRef(null);
```
Comment (kavinvenkatachalam) [inline frontend/src/_components/OrganizationManager/CreateOrganization.jsx:12]:
```suggestion
```

File: frontend/src/_components/OrganizationManager/CreateOrganization.jsx [hunk 3]
```diff
@@ -78,6 +87,7 @@ export const CreateOrganization = ({ showCreateOrg, setShowCreateOrg }) => {
             {t('globals.cancel', 'Cancel')}
           </ButtonSolid>
           <ButtonSolid
+            ref={createButtonRef}
```
Comment (kavinvenkatachalam) [inline frontend/src/_components/OrganizationManager/CreateOrganization.jsx:90]:
```suggestion
```

File: frontend/src/_components/OrganizationManager/CreateOrganization.jsx [hunk 4]
```diff
@@ -35,6 +36,13 @@ export const CreateOrganization = ({ showCreateOrg, setShowCreateOrg }) => {
     }
   };
 
+  const handleKeyDown = (e) => {
+    if (e.keyCode === 13) {
+      e.preventDefault();
+      createButtonRef.current.click();
```
Comment (kavinvenkatachalam) [inline frontend/src/_components/OrganizationManager/CreateOrganization.jsx:42]:
```suggestion
      createOrganization()
```

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@coderSuren Thanks for the PR.

Requested few changes. Please take a look and update the PR

**Comment by coderSuren** [comment]
@kavinvenkatachalam Thanks for your review, made the requested changes.

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**